docs/GettingStarted

```diff
@@ -165,6 +165,47 @@ The available settings are:
 * +trace_agent_hostname+: set the hostname of the trace agent.
 * +trace_agent_port+: set the port the trace agent is listening on.
 
+=== Sinatra
+
+The Sinatra integration traces requests and template rendering. The
+integration is based on the +Datadog::Contrib::Sinatra::Tracer+ extension.
+
+We currently support Sinatra >= 1.4.0.
```

Comment:
what are the most commonly used versions? let's try to support versions released in the last year.